powerpoint/How-to/work-with-partial-documents.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Work with Partial Documents
+keywords: vbapp10.chm583138
+f1_keywords:
+- vbapp10.chm583138
+ms.prod: powerpoint
+ms.date: 08/02/2022
+ms.author: ononder
+ms.localizationpriority: medium
+---
+
+
+# Work with Partial Documents
+
+When you open a presentation with content that is large in size, PowerPoint may serve the document in parts as partial documents. This allows for documents to be opened, edited, and collaborated upon quickly, while the larger media parts (e.g., videos), continue to load in the background. Similarly, since media is handled separately from the rest of the document, collaboration is smoother when media is inserted during a collaboration session.
+
+Because certain content can be deferred initially, some actions can't be taken on that content until the deferred content (e.g., video) is loaded. Additionally, there are certain actions like Save As, Export to Video, etc. that won’t function until all the deferred content are downloaded. User initiated operations will display UI informing the user of download progress, but that’s not possible for programmatic operations.  If you programmatically attempt to call an API to execute an action in these cases, it will fail with the following error message.
+
+
+```
+Run-time error '-2147188128 (80048260)':
+<object> (unknown member) : This method isn't supported until the presentation is fully downloaded. Visit this URL for more information: https://go.microsoft.com/fwlink/?linkid=2172228
+```
+
+
+## Understanding the Fully Downloaded State
```

Comment:
```suggestion
## Understand the fully downloaded state
```
Newer guidelines recommend you don't gerundize the leading verb in a heading. This is for localization and SEO. There's quite a lot of gerunding verbs in headings in msft docs, but I change them as I see them and fix in all new content.